regression/contracts/assert_ensures_01/test.desc

```diff
@@ -0,0 +1,7 @@
+CORE
+main.c
+--assert-ensures min
+^EXIT=10$
+^SIGNAL=0$
+^VERIFICATION FAILED$
```

Comment:
Could fail for other reasons -- suggest checking the particular failure line you're expecting, and/or comparing against a companion test that doesn't use `--assert-ensures` and for which verification succeeds